apps/core/scenes/passengerForm/BaggageBundles/BaggageBundle.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+
+import React from 'react';
+import { View } from 'react-native';
+import { StyleSheet, RadioButton } from '@kiwicom/universal-components';
+import { Text } from '@kiwicom/margarita-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import BaggageTypeDetail from './BaggageTypeDetail';
+
+type Props = {|
+  +price: ?string,
+  +bags: any,
+  +bagIndex: number,
+  +onToggleCheck: any,
+  +type: string,
+  +currentlyChosen: number,
+|};
+
+type State = {|
+  checked: boolean,
+|};
+
+export default class BuggageBundle extends React.Component<Props, State> {
```

Comment:
I believe it should be `BaggageBundle`, not `BuggageBundle` 😄

Reply:
Yes, stupid typo 😓